standalone/inc/pup/PUPScreen.cpp

```diff
@@ -79,7 +79,12 @@ PUPScreen::PUPScreen(PUP_SCREEN_MODE mode, int screenNum, const string& szScreen
    m_pPageTimer->SetElapsedListener(std::bind(&PUPScreen::PageTimerElapsed, this, std::placeholders::_1));
    m_pParent = nullptr;
 
-   LoadPlaylists();
+   for (const PUPPlaylist* pPlaylist : playlists) {
+      // make a copy of the playlist
+      PUPPlaylist *pPlaylistCopy = new PUPPlaylist(*pPlaylist);
```

Comment:
Just keeping things as the old code, not sure if this copy is 100% required, depends on if playlists are mutated during gameplay.